clients/fides-js/src/lib/gpp/constants.ts

```diff
@@ -1 +1,356 @@
-export const SUPPORTED_APIS = ["2:tcfeuv2"];
+import {
+  UsCaV1,
+  UsCaV1Field,
+  UsCoV1,
+  UsCoV1Field,
+  UsCtV1,
+  UsCtV1Field,
+  UsNatV1,
+  UsNatV1Field,
+  UsUtV1,
+  UsUtV1Field,
+  UsVaV1,
+  UsVaV1Field,
+} from "@iabgpp/cmpapi";
+
+interface GppMechanismField {
+  field: string;
+  not_available: number | number[];
+  opt_out: number | number[];
+  not_opt_out: number | number[];
+}
+interface GppFields {
+  gpp_notice_fields: string[];
+  gpp_mechanism_fields: GppMechanismField[];
+}
+type FidesRegionToGppFields = Record<string, GppFields>;
+type NoticeKeyToFidesRegionGppFields = Record<string, FidesRegionToGppFields>;
+
+/**
+ * Eventually this will be encoded on the privacy notice itself
+ *
+ * Currently based off of this table
+ * https://ethyca.atlassian.net/wiki/spaces/EN/pages/2852945989/2023-11-30+Fides.js+GPP+implementation?focusedCommentId=2861989902
+ */
+export const NOTICE_KEY_TO_FIDES_REGION_GPP_FIELDS: NoticeKeyToFidesRegionGppFields =
```

Comment:
@mfbrown would you be able to review this JSON? it's meant to be structured like an extension of a privacy notice. let me know if it makes sense, or if you'd like me to walk through it with you?